teamcity: failed tests on master: testrace (proposer evaluated kv)/TestLogicDistSQL, testrace (proposer evaluated kv)/TestLogicDistSQL/testdata/aggregate, test/TestReportUsage, test (proposer evaluated kv)/TestLogicDistSQL, test (proposer evaluated kv)/TestLogicDistSQL/testdata/aggregate

The following tests appear to have failed:

[#177630](https://teamcity.cockroachdb.com/viewLog.html?buildId=177630):

```
--- FAIL: testrace (proposer evaluated kv)/TestLogicDistSQL (177.420s)
null
--- FAIL: testrace (proposer evaluated kv)/TestLogicDistSQL/testdata/aggregate (1.640s)
logic_test.go:1503: 
	testdata/aggregate:760: 
	expected:
	    9.00 4.5 6.333333333333333
	    
	but found (query options: "trimdecimals") :
	    9.00  4.5  6.333333333333334  
	
logic_test.go:1533: testdata/aggregate:765: too many errors encountered, skipping the rest of the input
------- Stdout: -------
--- new user: root
--- queries start here
--- done: testdata/aggregate: 121 tests, 1 failures
--- FAIL: test/TestReportUsage (0.120s)
updates_test.go:150: expected node keys at least 90 got 84
updates_test.go:161: expected at least 90 keys in store 1 got 84
--- FAIL: test (proposer evaluated kv)/TestLogicDistSQL (35.750s)
null
--- FAIL: test (proposer evaluated kv)/TestLogicDistSQL/testdata/aggregate (0.290s)
logic_test.go:1503: 
	testdata/aggregate:760: 
	expected:
	    9.00 4.5 6.333333333333333
	    
	but found (query options: "trimdecimals") :
	    9.00  4.5  6.333333333333334  
	
logic_test.go:1533: testdata/aggregate:765: too many errors encountered, skipping the rest of the input
------- Stdout: -------
--- new user: root
--- queries start here
--- done: testdata/aggregate: 121 tests, 1 failures

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/testdata/aggregate
+++ b/pkg/sql/testdata/aggregate
@@ -758,12 +758,12 @@ EXPLAIN (EXPRS) SELECT MAX(x) FROM xyz WHERE (z, y) = (3.0, 2)
 2                 limit        1
 
 query RRR trimdecimals
-SELECT VARIANCE(x), VARIANCE(y::decimal), VARIANCE(z) FROM xyz
+SELECT VARIANCE(x), VARIANCE(y::decimal), round(VARIANCE(z), 14) FROM xyz
 ----
-9.00 4.5 6.333333333333333
+9.00 4.5 6.33333333333333
 
 query ITTT
-EXPLAIN (DEBUG) SELECT VARIANCE(x), VARIANCE(y::decimal), VARIANCE(z) FROM xyz
+EXPLAIN (DEBUG) SELECT VARIANCE(x), VARIANCE(y::decimal), round(VARIANCE(z), 14) FROM xyz
 ----
 0  /xyz/primary/1    NULL                            PARTIAL
 0  /xyz/primary/1/y  2                               PARTIAL
@@ -773,7 +773,7 @@ EXPLAIN (DEBUG) SELECT VARIANCE(x), VARIANCE(y::decimal), VARIANCE(z) FROM xyz
 1  /xyz/primary/4/z  6.0                             BUFFERED
 2  /xyz/primary/7    NULL                            PARTIAL
 2  /xyz/primary/7/z  8.0                             BUFFERED
-0  0                 (9.00, 4.5, 6.333333333333333)  ROW
+0  0                 (9.00, 4.5, 6.33333333333333)   ROW
 
 query R
 SELECT VARIANCE(x) FROM xyz WHERE x = 10

